coralproject/talk
# Adding coral embed stream

## What does this PR do?

Adds `coral-embed-stream` to the newly structured `talk` repo.

## How do I test this PR?

1. `npm install`
2. From the repo root, run `npm run embed-start`
3. Dev server should start.
4. Comment stream should load with mock data.
5. Posts won’t work, and a few 404s are expected until we get the endpoints up.

## Notes

- I’ve left the original web pack in place for reference, I’m not certain what the vision is for a central webpack so wanted to keep that separate. If run independently, this webpack file will build to the target dist directory.
